app/survey_config/business_config.py

```diff
@@ -60,3 +67,10 @@ def get_service_links(
             if is_authenticated
             else None
         )
+
+    @property
+    def stripped_base_url(self) -> str:
+        warn(
+            "base_url contains extra pathing which will eventually be corrected and this function will need to be removed"
+        )
+        return self.base_url.replace("/surveys/todo", "")
```

Comment:
Functionally, works as expected. Minor suggestion. Any reason not to overwrite the `base_url` with the stripped version as this approach doesn't prevent the next dev using `self.base_url` directly?.

Reply:
The options for the change were: _This can be done either for the self.base_url or for each of the resulting 3 URLs that are constructed_

I went for the second option, I figured the implications of only updating the 3 urls were considered already, but I guess if the base_url is likely to be wrong for a while it is worth doing then

Reply:
Ya, I think it's worth changing, we don't know when it will be fixed and turns as other urls were broken also.